lingetic-nextjs-frontend/utilities/testing-helpers.tsx

```
@@ -1,6 +1,22 @@
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { render } from "@testing-library/react";
 
+export function mockAudio() {
+  class AudioMock {
+    src: string = "";
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)